spec/request/folders_request_spec.rb

```diff
@@ -139,55 +139,64 @@
     describe 'nested resources' do
       it 'gets messages#index' do
         VCR.use_cassette('sm_client/folders/nested_resources/gets_a_collection_of_messages') do
-          get "/v0/messaging/health/folders/#{inbox_id}/messages"
+          VCR.use_cassette('sm_client/folders/gets_a_collection_of_folders') do
+            get "/v0/messaging/health/folders/#{inbox_id}/messages"
+          end
         end
```

Comment:
My changes to `client.rb` above seem to be working, but due to the additional `get_folders` call, it is messing up these pagination related unit tests.  I think I'm messing up somewhere with the nesting of the VCR cassettes. Could use some extra ey